ddtrace/tracer.py

```diff
@@ -167,14 +170,33 @@ def configure(self, enabled=None, hostname=None, port=None, uds_path=None, dogst
         if wrap_executor is not None:
             self._wrap_executor = wrap_executor
 
-        if collect_metrics and self._runtime_worker is None:
-            self._dogstatsd_host = dogstatsd_host or self._dogstatsd_host
-            self._dogstatsd_port = dogstatsd_port or self._dogstatsd_port
-            # start dogstatsd client if not already running
-            if not self._dogstatsd_client:
-                self._start_dogstatsd_client()
+        if collect_metrics is not None:
+            # We got a configure request, stop our worker if it exists
+            if self._runtime_worker:
+                self._runtime_worker.stop()
+                self._runtime_worker.join()
+                self._runtime_worker = None
 
-            self._start_runtime_worker()
+            # Store the new value
+            if collect_metrics is True:
+                self._collect_metrics_interval = RuntimeWorker.FLUSH_INTERVAL  # default interval
```

Comment:
we actually don't want people to configure this. It should probably only ever be `FLUSH_INTERVAL`.